packages/renderer/src/lib/onboarding/Onboarding.svelte

```diff
@@ -1,3 +1,22 @@
+<style>
+#stepBody::-webkit-scrollbar {
+  width: 1em;
+}
+#stepBody::-webkit-scrollbar-track {
+  -webkit-box-shadow: inset 0 0 6px rgba(0, 0, 0, 0.3);
+}
+#stepBody::-webkit-scrollbar-thumb {
+  background-color: #5c5c5c;
+}
+#stepBody::-webkit-scrollbar-thumb:hover {
+  background-color: #707073;
```

Comment:
I guess we can't add colors like that in svelte files.

it should be part of our color palette/being referenced